<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>view</class>
 <widget class="QMainWindow" name="view">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>551</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>731</width>
    <height>551</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>731</width>
    <height>551</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>0</y>
      <width>211</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Maze</string>
     </attribute>
     <widget class="QPushButton" name="save_maze">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>470</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="download_maze">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>430</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Загрузить файл</string>
      </property>
     </widget>
     <widget class="QPushButton" name="generation_maze">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Сгенерировать</string>
      </property>
     </widget>
     <widget class="QLabel" name="width_maze">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>3</y>
        <width>58</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Ширина</string>
      </property>
     </widget>
     <widget class="QLabel" name="hight_maze">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>58</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Высота</string>
      </property>
     </widget>
     <widget class="QPushButton" name="maze_way">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Решение</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Координаты старта</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Координаты конца</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>58</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Х</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>58</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Х</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>270</y>
        <width>58</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>У</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>210</y>
        <width>58</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>У</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cols_maze">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>50</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>25</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="rows_maze">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>50</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>25</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
     </widget>
     <widget class="QSpinBox" name="x_start">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>210</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>49</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="y_start">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>210</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>49</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="x_finish">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>270</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>49</number>
      </property>
      <property name="value">
       <number>49</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="y_finish">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>270</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>49</number>
      </property>
      <property name="value">
       <number>49</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Cave</string>
     </attribute>
     <widget class="QLineEdit" name="miliseconds">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>290</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>1000</string>
      </property>
     </widget>
     <widget class="QLabel" name="life">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>101</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Предел жизни</string>
      </property>
     </widget>
     <widget class="QPushButton" name="download_cave">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>430</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Загрузить файл</string>
      </property>
     </widget>
     <widget class="QPushButton" name="save_cave">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>470</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Пошаговый режим</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="hight_cave">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>42</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Высота</string>
      </property>
     </widget>
     <widget class="QPushButton" name="start">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string> Старт</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>250</y>
        <width>99</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Таймер</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>99</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>По кнопке</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="dead">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>101</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Предел смерти</string>
      </property>
     </widget>
     <widget class="QPushButton" name="next_step">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Следующий шаг</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>14</x>
        <y>290</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Миллисекунд</string>
      </property>
     </widget>
     <widget class="QLabel" name="width_cave">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>13</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Ширина</string>
      </property>
     </widget>
     <widget class="QPushButton" name="generation_cave">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Сгенерировать</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cols_cave">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>0</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>100</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>50</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>25</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="rows_cave">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>100</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>50</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>25</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>20</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
     </widget>
     <widget class="QSpinBox" name="limit_life">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>7</number>
      </property>
      <property name="value">
       <number>4</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="limit_dead">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>110</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="maximum">
       <number>7</number>
      </property>
      <property name="value">
       <number>4</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>501</width>
      <height>501</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>501</width>
      <height>501</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>501</width>
      <height>501</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
